apitrackingmore.py

```diff
@@ -20,6 +34,7 @@ def get_or_create_tracking_item(carrier, code):
         trackingmore.create_tracking_item(tracking_data)
         tracking_data = trackingmore.get_tracking_item(carrier, code)
     except trackingmore.trackingmore.TrackingMoreAPIException as e:
+        utils.log(logger_error, "{e.err_code} \t{e.err_type} \t{e} \t({carrier}, {code})".format(e=e, carrier=carrier, code=code))
```

Comment:
Melhor forma de criar a mensagem de log é usando `%s`,  `%r`, etc, ao invés de `.format()` ou `f""`. Estranho isso, né? Mas a explicação é que, se usarmos ferramentas de agregação de logs, é possível agrupar as excessões que usamo o mesmo template. Por exemplo, se tivermos o log abaixo:

```python
logger.error("Code %s not found for carrier %s", code, carrier) 
```

as ferramentas, em especial o Sentry, conseguem agrupar todas os logs em que o template é `Code %s not found for carrier %s"`. Se usarmos `format`, elas não conseguiriam identificar, por exemplo, que `Code AB012345678 not found for carrier correios"` é do "mesmo grupo" que `Code YZ098765432 not found for carrier correios"`.

Essa agregação ajuda muito na hora de debugar possíveis bugs.

Reply:
Interessante! Eu não fazia ideia hahah
A interpolação das variáveis na `string` é feita pela ferramenta então?